applications/common/frontend/src/components/Quote/Quote.scss

```diff
@@ -0,0 +1,32 @@
+/*!
+ * <!--
+ *     Copyright (C) 2024 Dynamic Solutions
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ * -->
+ */
+
+@use 'src/atomic-design-system/02-molecule/m.quote' as m-quote;
+
+.quote {
+
+  @include m-quote.m-quote;
+
+  .quote-text {
+    @include m-quote.m-quote-text;
+  }
+
+  .quote-author {
+    @include m-quote.m-quote-author;
```

Comment:
Whole quote is a molecule, but mixins you used are not molecules, they're atoms (cannot be any smaller, nor divided into smaller parts). If we stick to atomic design I would move quote text and quote autor to atoms, then in mixin quote use them, and here should be included only quote molecule.